policyengine_us/variables/gov/states/ri/tax/income/exemption/ri_exemptions.py

```diff
@@ -0,0 +1,34 @@
+from policyengine_us.model_api import *
+
+
+class ri_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Rhode Island exemptions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022%20Tax%20Rate%20and%20Worksheets.pdf"
+    defined_for = StateCode.RI
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.ri.tax.income.exemption
+
+        exemptions = tax_unit("exemptions", period)
+
+        exemption_amount = exemptions * p.amount
+
+        agi = tax_unit("ri_agi", period)
```

Comment:
is it AGI or modified AGI (as the parameter description states)?

Reply:
It's modified AGI. The modified AGI has not been merged yet, so we use a placement variable to represent it.

Reply:
Do they call it Rhode Island AGI or Rhode Island modified AGI? In some states these are separate concepts, so let's describe it consistently with the law/form.

Reply:
The legal code mentions "taxpayer's adjusted gross income", and the worksheet says "Enter your modified federal AGI". In the Rhode Island AGI PR, I will name the label to the correct name regarding to the tax form!

Reply:
Thanks, I see the form points to RI-1040 line 3 which indicates just one modified (RI) AGI concept: https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022_1040WE_w_0.pdf

Could you mention this in a comment?

Reply:
Ok! Thanks!